hadoop-ozone/ozonefs/src/main/java/org/apache/hadoop/fs/ozone/OzoneFSInputStream.java

```diff
@@ -76,4 +79,21 @@ public boolean seekToNewSource(long targetPos) throws IOException {
   public int available() throws IOException {
     return inputStream.available();
   }
+
+  /**
+   * @param buf the ByteBuffer to receive the results of the read operation.
+   * @return the number of bytes read, possibly zero.
+   * @throws IOException if there is some error performing the read
+   */
+  @Override
+  public int read(ByteBuffer buf) throws IOException {
+
+    int bufInitPos = buf.position();
+    while (buf.hasRemaining() && (inputStream.available() > 0)) {
```

Comment:
Instead of reading byte by byte, can we use inputStream.read(byte b[], int off, int len)?

Something in similar terms
```
    int readLen = Math.min(buf.remaining(), inputStream.available());
    byte[] readData = new byte[readLen];

    inputStream.read(readData, bufInitPos, readLen);
    buf.put(readData);
```

Reply:
Thanks @bharatviswa504 for the review.
Fixed, and work well.